ui/component/channelSelector/view.jsx

```diff
@@ -26,8 +26,11 @@ type Props = {
   storeSelection?: boolean,
   doSetDefaultChannel: (claimId: string) => void,
   isHeaderMenu?: boolean,
+  isPublishMenu?: boolean,
+  isTabHeader?: boolean,
   autoSet?: boolean,
   channelToSet?: string,
+  disabled?: boolean,
 };
 
 export default function ChannelSelector(props: Props) {
```

Comment:
_This is also happening in production, probably started since the default/active channel changes.  If it's a big fix, can ignore and file separately._

:exclamation: If you have an anonymous Post and tried to edit, the channel selector is pointing to the current active channel rather than anonymous.  If unnoticed, the user ends up publishing using a channel.